pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -1026,6 +1037,86 @@ func registriesConfigIgnition(templateDir string, controllerConfig *mcfgv1.Contr
 	return &registriesIgn, nil
 }
 
+func (ctrl *Controller) createDefaultContainerRuntimeMC() error {
+	// Check if the crio-default-container-runtime config map exists in the openshift-machine-config-operator namespace
+	defaultContainerRuntimeCM, err := ctrl.kubeClient.CoreV1().ConfigMaps(ctrl.namespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
```

Comment:
Theoretically we try to avoid any direct Get() calls in any of our syncs, as in theory they could trigger and hit API limits.

Practically speaking though, random syncs for containerruntimeconfig objects should not happen very often (~every 10 minutes or so in steady state?) so this is probably fine. It'd be better if we can use a lister, but since this is 4.17 only, I'm fine if we want to keep as is

Reply:
Discussed this off-line, I'm gonna keep this as-is unless CI quotas are hit.